tests/playwright-test/basic.spec.ts

```diff
@@ -251,3 +251,103 @@ test('should work with default export', async ({ runInlineTest }) => {
   expect(result.passed).toBe(1);
   expect(result.failed).toBe(0);
 });
+
+test('should work with test wrapper', async ({ runInlineTest }) => {
+  const result = await runInlineTest({
```

Comment:
How did this fail before?

Reply:
"2 passed, 2 skipped" instead of "4 passed"